api/docs/docs/data-definitions.md

```diff
@@ -237,4 +260,42 @@ number of people vaccinated with both the first and second dose.
 * JSON file fields: ``actuals.deaths``, ``actualsTimeseries.*.deaths``
 
 
+## Community Levels
+
+  Community level for region.
+
+  See https://www.cdc.gov/coronavirus/2019-ncov/science/community-levels.html
+  for details about how the Community Level is calculated and should be
+  interpreted.
+
+  The values correspond to the following levels:
+
+  | API value  | Community Level |
+  | ---------- | --------------- |
+  | 0 | Low |
+  | 1 | Medium |
+  | 2 | High |
+
+  Note that we provide two versions of the Community Level. One is called
+  `canCommunityLevel` which is calculated using CAN's data sources and is
+  available for states, counties, and metros. It is updated daily though
+  depends on hospital data which may only update weekly for counties. The
+  other is called `cdcCommunityLevel` and is the raw Community Level published
+  by the CDC. It is only available for counties and is updated on a weekly
+  basis.
+
+**Where to access**  
+* CSV column names: ``communityLevels.canCommunityLevel``
+* JSON file fields: ``communityLevels.canCommunityLevel``, ``communityLevelsTimeseries.*.canCommunityLevel``
+
+  and 
+
+* CSV column names: ``communityLevels.cdcCommunityLevel``
+* JSON file fields: ``communityLevels.cdcCommunityLevel``, ``communityLevelsTimeseries.*.cdcCommunityLevel``
+
+
+## Health Service Areas (HSAs)
+
+  An HSA is a collection of one or more contiguous counties which are relatively self-contained with respect to hospital care. HSAs are used when calculating county-level hospital metrics in order to correct for instances where an individual county does not have any, or has few healthcare facilities within its own borders. For more information see https://seer.cancer.gov/seerstat/variables/countyattribs/hsa.html.
 
+  The source for our county to HSA mappings is [`data/misc/cdc_hsa_mapping.csv`](https://github.com/covid-projections/covid-data-model/blob/main/data/misc/cdc_hsa_mapping.csv) which follows the HSA definitions used by the CDC in their [COVID-19 Community Levels](https://www.cdc.gov/coronavirus/2019-ncov/your-health/covid-by-county.html). HSA populations are calculated as the sum of the component county populations.
```

Comment:
You could probably drop `data/misc` from ``[`data/misc/cdc_hsa_mapping.csv`]`` since that path is only really relevant to us.  I'd also change the target to be the raw CSV file: https://raw.githubusercontent.com/covid-projections/covid-data-model/main/data/misc/cdc_hsa_mapping.csv